xdn-sapper-example/src/routes/_error.svelte

```diff
@@ -0,0 +1,40 @@
+<script>
```

Comment:
You should convert this file from tabs to spaces to be consistent with the rest of the sources.